pages/api/transactions/[address].ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiResponse, NextApiRequest } from 'next/types'
+import { getAddress } from 'services/bchdService'
+import { RESPONSE_MESSAGES } from 'constants/index'
+
+const { ADDRESS_NOT_PROVIDED_400 } = RESPONSE_MESSAGES
+
+export default async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+  try {
+    const { address } = req.query
```

Comment:
I think is best to wrap all of this inside a `if (req.method == 'GET') {` so the endpoint ignores all other kinds of requests